util/ui.ts

```diff
@@ -97,10 +97,12 @@ export const getActiveUserTypes = (): {
  *
  * Able to handle all the date/time formats that are generated by the CDP.
  */
-export const getDateFromCDPFileName = (filename: string): string => {
-  const TRIP_DATA_SUFFIX = '-anon-trip-data'
-  const date = filename.split(TRIP_DATA_SUFFIX)?.[0].split('/')?.[1]
-  if (!date || !filename.includes(TRIP_DATA_SUFFIX)) return filename
+export const getDateFromCDPFileName = (
+  filename: string,
+  tripDataSuffix: string
+): string => {
+  const date = filename.split(tripDataSuffix)?.[0].split('/')?.[1]
+  if (!date || !filename.includes(tripDataSuffix)) return filename
```

Comment:
```suggestion
  if (!date || !filename.endsWith(tripDataSuffix)) return filename
```

Reply:
I think the string technically ends with `.zip`!